src/components/ViewStylePropTypes.js

```diff
@@ -1,2 +1,86 @@
 // @flow
-export default {};
+import * as PropTypes from 'prop-types';
+
+const BorderStyle = PropTypes.oneOf(['solid', 'dotted', 'dashed']);
+const Color = PropTypes.oneOfType([PropTypes.string, PropTypes.number]);
+const Overflow = PropTypes.oneOf(['visible', 'hidden', 'scroll']);
+
+export default {
+  color: Color,
+  shadowColor: Color,
+  shadowInner: PropTypes.bool,
+  shadowSpread: PropTypes.number,
+  shadowOffset: { width: PropTypes.number, height: PropTypes.number },
```

Comment:
That's not a proper PropTypes.
```suggestion
  shadowOffset: PropTypes.shape({
    width: PropTypes.number,
    height: PropTypes.number
  }),
```